app/pb_kits/playbook/pb_date_picker/sass_partials/_overrides.scss

```diff
@@ -0,0 +1,37 @@
+// media query non mobile styles
+@media (min-width: 500px) {
+  .flatpickr-calendar {
+    padding: 8px;
+    width: 323.88px
+  }
+  .dayContainer {
+    width: 315.88px
+  }
+  .flatpickr-months {
+    margin-left: -8px;
+    margin-top: -8px;
```

Comment:
Only way I could get things re-aligned after adding padding.  Tried many other solutions.  Tricky because I had to work with whatever divs flatpickr already had templated instead of being able to rearrange them myself.